lib/new.cc

```diff
@@ -191,7 +215,7 @@ NAN_METHOD(WrappedRE2::New) {
 		return Nan::ThrowTypeError("Expected string, Buffer, RegExp, or RE2 as the 1st argument.");
 	}
 
-	if (needConversion && data && size && translateRegExp(data, size, buffer)) {
+	if (needConversion && translateRegExp(data, size, buffer)) {
```

Comment:
I removed the check for `data` here because it is unnecessary (checked 4 lines earlier), and the check for `size` because `translateRegExp` now has something to do when `!size`.